### ISSUE TITLE
AwsTileRequest.get_data() doesn't work

### ISSUE THREAD

**Issue by shmulik90** [sentinel-hub/sentinelhub-py#33]
Hi there,
I'm jsut getting into your python tool but I am not able to reproduce your Aws example. 
![grafik](https://user-images.githubusercontent.com/38791059/40331391-d772b90e-5d15-11e8-96ff-bf463176a959.png)

**Comment by AleksMat** [comment]
Hi shmulik90,

you are right, the example doesn't work anymore and we found out the reason is the new version of `opencv-python` Python package used for reading JPEG 2000 images from file. 

It was updated 2 days ago from `3.4.0.14` to `3.4.1.15` and now supports OpenCV library `3.4.1`. The new version can only read 8-bit or 16-bit JPEG 2000 files and in case of Sentinel-2 tiles, which are 15-bit, just returns `None`.
(This is actually somewhat improvement from the previous version which read 15-bit incorrectly and we had to write additional code in `sentinelhub` to correct the values.)

So the solution would be that you revert your version of `opencv-python` library to `3.4.0.14` with:
```bash
pip uninstall opencv-python
pip install opencv-python==3.4.0.14
```
And we will update `sentinelhub` setup with dependency to this older version of `python-opencv`. Until OpenCV is improved again or we find alternative library for reading 15-bit jp2 files this solution will have to do.
